# OpenVins publishes velocities in body frame (some other algorithms don't)
# When false, republisher will transform velocities in global (vins, navigation, world) frame into body frame
velocity_in_body_frame: true

# Subtract the first heading from all msgs so that the origin is in zero (OpenVINS sometimes starts rotated by pi/2)
init_in_zero: true

# Wait for a service call that calibrates the level horizon (zero roll and pitch)
# VINS algorithms initialize during initial motion, so the origin can be not gravity-aligned
compensate_initial_tilt: false

# limits the processing to this rate (the real rate will be typically slightly lower)
# disable to keep processing at camera FPS
rate_limiter:
  enabled: false
  max_rate: 100.0 # [Hz]
